lib/modules/dosomething/dosomething_campaign/includes/Campaign.php

```diff
@@ -160,6 +160,7 @@ private function build($data, $display = 'teaser') {
 
         $this->scholarship = $this->getScholarship();
         $this->staff_pick = $this->getStaffPickStatus();
+        $this->competition = $this->getCompetition($this->id);
```

Comment:
this won't still work since the current Campaign object `$this` no longer has that method. you can call the method to get competitions in the **dosomething_campaign.query.inc` something along the normal Drupal naming scheme, like `dosomething_campaign_is_competion($id)` and then you call this from the object:

```
$this->competition = dosomething_campaign_is_competition($this->id);
``